app/models/dialog_field.rb

```diff
@@ -41,6 +41,9 @@ class DialogField < ApplicationRecord
 
   after_initialize :default_resource_action
 
+  supports(:update) { _("Can not edit read only model") if read_only? }
+  supports(:delete) { _("Can not delete read only model") if read_only? }
```

Comment:
Strange that we have this for Dialog Field, but not other Dialog models?

Reply:
So lets just skip on this one?